tests/integration/targets/dcnm_vrf/tests/dcnm/overridden.yaml

```diff
@@ -201,12 +201,8 @@
     - 'result.response[0].RETURN_CODE == 200'
     - 'result.response[1].RETURN_CODE == 200'
     - '(result.response[0].DATA|dict2items)[0].value == "SUCCESS"'
-    - '(result.response[0].DATA|dict2items)[1].value == "SUCCESS"'
```

Comment:
Remind me why we are deleting this check?

Reply:
There was an issue in vrf module, in the following scenario. If a switch is already attached to a vrf and the same switch is being attached again along with another switch to same vrf, then the attach was sent to both the switches. The attach request was being sent for switch which is already attached, which ideally should not have happened. The IT cases were written to check for 2 attach in response. The attached should have been sent only for the new switch being added. This works fine for network module(attach is sent for newly added switch only). There was an issue in vrf module code while determining the switches which are already attached. Have fixed this issue now, and after the fix the attach is being sent only for the newly added switch and I have removed the check for second attach response in IT cases as it will not be there.